site/docs/components/drawer/usage.mdx

```diff
@@ -9,28 +9,41 @@ data:
 
 ### Using the component
 
-Use the component by wrapping it around your content and setting the desired position. Provide a mechanism, such as [`Button`](../button), for users to open and close it.
+Drawers are always modal. When open, the drawer displays a visual overlay, a [scrim](/salt/components/scrim) component, over the underlying content.
 
-When open, the drawer displays a visual overlay, also known as a scrim, over the underlying content. It also plays relevant animations when opening and closing that can be visually customized via CSS.
+Always set a logical width suitable for the content inside the drawer. Refer to Salt's [Forms](/salt/patterns/forms) pattern for information on logical forms width, and the [typography foundation](/salt/foundations/typography) for guidance on paragraph text width.
 
-Use the `useDrawer` hook to handle interactions with the drawer, such as opening and closing it.
+#### Best practices
+
+- Use top drawer to display information best displayed in horizontal format, e.g., complex filters or a summary panel.
+- Use bottom drawer for any master-detail layout containing graphs and images.
+- Use right drawer to show supporting information, notifications or quick settings.
```

Comment:
@mikearildbrown  i added this section in usage, it's the same as what's written in example "Positon" 's description (tagged you where it sits) - which of these 2 locations would be best suited to keep this info?

Reply:
The same points already appear twice in examples section so think they can probably be removed here

Reply:
removed this content in favour of having it under the position examples